docs/content/en/docs/pipeline/chain-security.md

```diff
@@ -0,0 +1,370 @@
+```
+
+With these steps completed, the preliminary setup is ready, paving the way for secure and efficient pipeline operations for supply chain security.
+
+## Triggering the Test Pipeline
+
+### Creating Kurator Pipeline Test Example
+
+Create a pipeline that includes an image task using the following command:
+
+```console
+echo 'apiVersion: pipeline.kurator.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: quick-start
+  namespace: kurator-pipeline
+spec:
+  description: "this is a quick-start pipeline, it shows how to use customTask and predefined Task in a pipeline"
+  tasks:
+    - name: git-clone
+      predefinedTask:
+        name: git-clone
+        params:
+          git-secret-name: git-credentials
+    - name: cat-readme
+      customTask:
+        image: zshusers/zsh:4.3.15
+        command:
+          - /bin/sh
+          - -c
+        args:
+          - "cat $(workspaces.source.path)/README.md"
+    - name: go-test
+      predefinedTask:
+        name: go-test
+        params:
+          packages: ./...
+    - name: go-lint
+      predefinedTask:
+        name: go-lint
+        params:
+          packages: "./..."
+          flags: "--disable=errcheck,unused,gosimple,staticcheck --verbose --timeout 10m"
+    - name: build-and-push-image
+      predefinedTask:
+        name: build-and-push-image
+        params:
+          image: "ghcr.io/<username>/<image-name>"'| kubectl apply -f -
+```
+
+Replace `<username>` with your GitHub username and `<image-name>` with your image name, like `kurator-test:0.4.1`.
+
+### Exposing Services
+
+Similar to the previous pipeline, expose the services automatically created by this pipeline:
+
+```console
+kubectl port-forward --address 0.0.0.0 service/el-quick-start-listener 30002:8080 -n kurator-pipeline
+```
+
+### Configuring Webhook
+
+Set up a webhook for this pipeline as well, following the detailed process in [Setting Up Your Pipeline](https://kurator.dev/docs/pipeline/setting/).
+
+### Triggering the Pipeline
+
+To trigger the pipeline, consider pushing some content to the repository, such as a modification to the README file. 
+The information about the received event can be observed in the terminal where the port-forwarding service is running:
+
+```console
+Forwarding from 0.0.0.0:30002 -> 8080
+Handling connection for 30002
+```
+
+### Viewing Pipeline Execution Results
+
+After the pipeline is triggered, individual pods will be created for each task in the pipeline, executing them sequentially. 
+You can view the status of each task's execution with specific commands.
+
+Similarly, after obtaining the pipeline execution name from the command `kurator pipeline execution list -n kurator-pipeline --kubeconfig /root/.kube/kurator-host.config`, 
+you can view the execution logs of each task in the pipeline using the following command:
+
+```console
+$ kurator pipeline execution logs <pipeline-execution>  -n kurator-pipeline --tail 10 --kubeconfig /root/.kube/kurator-host.config
```

Comment:
can you provide a command to get this name

Reply:
`kurator pipeline execution list -n kurator-pipeline --kubeconfig /root/.kube/kurator-host.config`, 
I've already demonstrated this command earlier, and its usage has appeared in previous documents.
I can emphasize this command using console notation.